l10n_br_crm/models/crm_lead.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (C) 2012 - TODAY  Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from erpbrasil.base import misc
+    from erpbrasil.base.fiscal import cnpj_cpf, ie
+except ImportError:
+    _logger.error("Biblioteca erpbrasil.base não instalada")
+
+
+class Lead(models.Model):
+    """ CRM Lead Case """
+
+    _inherit = "crm.lead"
+
+    legal_name = fields.Char(
+        string="Legal Name", help="nome utilizado em documentos fiscais"
+    )
+
+    cnpj = fields.Char(string="CNPJ")
+
+    inscr_est = fields.Char(string="Inscr. Estadual/RG")
+
+    inscr_mun = fields.Char(string="Inscr. Municipal")
+
+    suframa = fields.Char(string="Suframa")
+
+    city_id = fields.Many2one(
+        comodel_name="res.city",
+        string="City ID",
+        domain="[('state_id', '=', state_id)]",
+    )
+
+    country_id = fields.Many2one(default=lambda self: self.env.ref("base.br"))
+
+    district = fields.Char(string="District")
+
+    street_name = fields.Char(string="Street Name")
```

Comment:
@renatonlima @rvalyi  tive que adicionar o street_name aqui senão dá erro na hora de abrir o formuário do Lead.
cc @netosjb

Reply:
Será se não teria que fazer essa mudança na v12 tb (confesso que eu não testei) ? Se for seria melhor fazer num commit separado para poder compartilhar ele entre à v12 e a v14.

Reply:
Boa.. vou separar o commit

Reply:
Feito